### PR TITLE
feat: add UniProt identifiers

### PR DESCRIPTION
**Pull request by cheng-yu-zhang**
### Main improvements in this PR:
All files and codes are in `yeast-GEM/add_uniprot_id`.

1. `get_uniprot_id.py` is for getting uniprot ids from the uniprot website.
2. `SGD_with_Uniprot.csv` is the output of `get_uniprot_id.py` with gene names and its uniprot ids.
3. `SGDgeneNames.tsv` is originally in `yeast-GEM/data/database`.
4. `addUniprotID.m` was used to add uniprot ids into yeast-GEM.

**I hereby confirm that I have:**

- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: model/yeast-GEM.yml [hunk 1]
```diff
@@ -110205,3589 +110205,5808 @@
...
+      - annotation: !!omap
+          - Uniprot ID: "P00424"
     - !!omap
       - id: "YNL054W"
       - name: "VAC7"
       - annotation: !!omap
-          - kegg.genes: "sce:YNL054W"
-          - ncbigene: "855673"
-          - ncbiprotein: "NP_014344"
-          - refseq: "NM_001182893"
```
Comment (feiranl) [inline model/yeast-GEM.yml:113017]: Not sure wether it is a good idea to remove all other annotations? Maybe an addition of Uniprot ID is better than replacement.

Comment (cheng-yu-zhang) [reply model/yeast-GEM.yml:113017]: I will recheck the codes. Maybe there are some mistake.

Comment (cheng-yu-zhang) [reply model/yeast-GEM.yml]: Problem solved

**Review by edkerk — APPROVED** [review]
I rebased the PR to `develop`, refactored the code to use RAVEN's `editMiriam` and removed duplicate `SGDgeneNames.tsv`. Otherwise, LGTM.

**Review by feiranl — APPROVED** [review]
LGTM!

**Pull request merged**